source/index-management.rst

```diff
@@ -682,6 +682,11 @@ Common API Components
      * server-side is used.
      */
     collation: Document;
+
+    /**
+     * Optionally specifies the wildcard projection of a wildcard index.
+     */
```

Comment:
Do you think it would be helpful to add an example `wildcardProjection` document to the doc string?

Reply:
Maybe - I don't think it's consistent, though, because other options like `storageEngine` don't have an example. Also, unless drivers are going to be doing any validation on the document, I don't see the value of having an example here. It seems like something that would be useful for users of drivers, but not driver authors.

Reply:
I figured the other options _would_ have example but since they don't this SGTM.

Reply:
I concur that this is something to sort out in our user documentation, but the spec need only concern itself with enforcing that the option is a document type.